<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#2e2e2e;</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Real-Time Sinusoidal Model Transformation</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>21</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider {
            background-color: #FF3B3B3B;
            color: white;
            border : 0.1em;
            height:  1.5em;
border-radius:7px;
        }

QSlider:hover {
       background-color:rgb(94, 92, 100);
       
}        </string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="listen_checkbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	color:white;
}

QCheckBox::indicator { 
	border: 1px solid; 
	border-radius:7px;
	border-color: white; }

QCheckBox::indicator:checked { 
background-color:#0096FF; 
}</string>
   </property>
   <property name="text">
    <string>Listen?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #0096FF;
	border-radius: 10px;
	color: white;

}


QPushButton:hover {
	background-color: #3063A5;

};</string>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="record_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(237, 51, 59);
	border-radius: 10px;
	color: white;

}


QPushButton:hover {
	
	background-color: rgb(192, 28, 40);

};</string>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </widget>
  <widget class="QLabel" name="red_border">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>961</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-color: rgb(192, 97, 203);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="recording_label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #2e2e2e;
}</string>
   </property>
   <property name="text">
    <string>Recording</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>red_border</zorder>
  <zorder>label_2</zorder>
  <zorder>verticalSlider</zorder>
  <zorder>label</zorder>
  <zorder>listen_checkbox</zorder>
  <zorder>reset_btn</zorder>
  <zorder>record_btn</zorder>
  <zorder>recording_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
